ci-operator/step-registry/baremetalds/packet/setup/baremetalds-packet-setup-commands.sh

```diff
@@ -6,9 +6,6 @@ set -o pipefail
 
 echo "************ baremetalds packet setup command ************"
 
-# TODO: Remove once OpenShift CI will be upgraded to 4.2 (see https://access.redhat.com/articles/4859371)
-~/fix_uid.sh
```

Comment:
I'm somehow surprised that this could work, last time we tried it was required to run correctly the Ansible playbook :thinking:

Reply:
Maybe because ansible is working against localhost it doesn't actually SSH?

Reply:
IINW there was an issue even on localhost. Maybe it's because we're running on a upgraded cluster (>4.2)